### PR TITLE
feat(worker): Add Bounds extractor

### PR DESCRIPTION
**Pull request by dongri**
# Overview
Add BoundsExtractor function

## Summary by CodeRabbit

- **New Features**
  - Introduced the "BoundsExtractor" processor for extracting bounds from various geometries in 3D space, enhancing spatial data processing capabilities.
  - Added comprehensive workflow configurations for the "BoundsExtractor" to facilitate seamless integration and usage.

### CODE REVIEW

**Comment by miseyu** [comment]
@dongri 
There are conflicts, please resolve the conflicts.

File: worker/crates/action-processor/src/geometry/bounds_extractor.rs [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    fn parameter_schema(&self) -> Option<schemars::schema::RootSchema> {
+        None
+    }
+
+    fn categories(&self) -> &[&'static str] {
+        &["Geometry"]
+    }
+
+    fn get_input_ports(&self) -> Vec<Port> {
+        vec![Port::new("lod1")]
```
Comment (miseyu) [inline worker/crates/action-processor/src/geometry/bounds_extractor.rs:72]:
```suggestion
        vec![DEFAULT_PORT.clone()]
```

File: worker/crates/action-processor/src/geometry/bounds_extractor.rs [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    }
+
+    fn get_output_ports(&self) -> Vec<Port> {
+        vec![DEFAULT_PORT.clone()]
```
Comment (miseyu) [inline worker/crates/action-processor/src/geometry/bounds_extractor.rs:76]:
```suggestion
        vec![DEFAULT_PORT.clone(), REJECTED_PORT.clone()]
```

File: worker/crates/action-processor/src/geometry/bounds_extractor.rs [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+            attributes.insert(Attribute::new("zmin"), bounds.min_z_value());
+            attributes.insert(Attribute::new("zmax"), bounds.max_z_value());
+
+            let feature = Feature {
+                attributes,
+                ..feature.clone()
+            };
+            fw.send(ctx.new_with_feature_and_port(feature, DEFAULT_PORT.clone()));
+        } else {
+            fw.send(ctx.new_with_feature_and_port(feature.clone(), DEFAULT_PORT.clone()));
```
Comment (miseyu) [inline worker/crates/action-processor/src/geometry/bounds_extractor.rs:131]:
```suggestion
            fw.send(ctx.new_with_feature_and_port(feature.clone(), REJECTED_PORT.clone()));
```

**Comment by miseyu** [comment]
@dongri 
Commented on. Please check it!!

**Comment by dongri** [comment]
@miseyu Thank you for review
Fixed

**Review by miseyu — APPROVED** [review]
Thank you so much!!
Please merge it.
LGTM

**Pull request merged**